src/util/saveArticles.ts

```diff
@@ -915,6 +925,15 @@ function isSubpage(id: string, mw: MediaWiki) {
     return false;
 }
 
+function addWebpScript(webp: boolean) {
```

Comment:
It is not OK to have a script called `addWebpScript()` which add empty string. In general you should try to not add, change function calls. The less the functions change, the less arguments they have, the better.